packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts

```diff
@@ -275,6 +275,12 @@ export class RelationshipDeclarationOperations {
         )}EdgeSort`;
     }
 
+    public get edgeAggregationWhereInputTypeName(): string {
+        return `${upperFirst(this.relationshipDeclaration.source.name)}${upperFirst(
```

Comment:
The `upperFirst` here is not consistent with the rest of the type names

Reply:
I know but they are inconstistent for all the edge types:

https://github.com/neo4j/graphql/blob/931782b4aeb943cf5951cbd692822f03b72a9ae7/packages/graphql/src/schema-model/relationship/model-adapters/RelationshipDeclarationOperations.ts#L247-L276

I think they all need to be  without the 1st  `upperFirst`,  what do you think @a-alle ?

Reply:
I create a separate PR for this